Python/Product/BuildTasks/Microsoft.PythonTools.Web.targets

```diff
@@ -94,30 +93,23 @@ permissions and limitations under the License.
     </ItemGroup>
   </Target>
 
+  <Target Name="CollectFilesFromIntermediateAssembly" />
```

Comment:
Just checking that this is needed, since it's new and doesn't appear to be doing anything.